src/test/java/com/EighthLight/app/TestConfig.java

```diff
@@ -103,8 +94,27 @@ void invalidBoardSizeSelectionDisplaysAnErrorMessage() {
     }
 
     @Test
-    void gameMode1StartsWithTwoHumans() {
+    void invalidPlayerOrderSelectionDisplaysAWarningAndPrompt() {
```

Comment:
This is the "sad" path for when a user chooses a player order which does not exist.